app/assets/javascripts/features/dropdowns.js

```diff
@@ -7,7 +11,7 @@ $(function() {
         ev.stopPropagation();
     });
 
-    $(document).on("click", ".ui-dropdown-menu .icon", function() {
+    $(document).on("click", ".ui-dropdown-menu:not(.search) .icon", function() {
```

Comment:
we should probably remove the `ui-dropdown-menu` class from the search button instead of adding a not selector